tools/checkstyle/pom.xml

```diff
@@ -25,12 +25,18 @@
     <version>4.18.0.0-SNAPSHOT</version>
 
     <properties>
+        <project.build.outputTimestamp>10</project.build.outputTimestamp>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.2</version>
```

Comment:
Should this version be defined as property in root pom.xml?

Reply:
it is used nowhere else, hence @hboutemy put it in the local pom.xml. I think it is alright in this case.